manager/manager.py

```diff
@@ -0,0 +1,1105 @@
+    machine for behavioural connection, machine for analysis for specific data etc.).
+
+    On first use on a new profile, the user will be prompted to set configurations with the function
+    make_config_file().
+
+    For transferring data between a remote data storage with SSH, use setup setup_ssh_connection_to_remote_server().
+    This will allow you to check the server Key, add host key to profile if accepted, and setup ssh key pair.
+
+    INPUTS: username - the username for the profile to use. Profile files are stored in your Appdir folder
+                       (platform specific). Use get_appdir_path() to retrieve the path.
+    """
+
+    def __init__(self, username: str):
+
+        self.username = username
+
+        self._config_path = self._join("appdir", "config.yaml")
+        self.cfg = self._attempt_load_configs(prompt_on_fail=True)
+
+        if self.cfg:
+            self._ssh_key_path = None
+            self._ses_folders = None
+
+            if self.cfg:
+                self.set_attributes_after_config_load()
+
+    def set_attributes_after_config_load(self):
+        """
+        Once config file is loaded, update all private attributes according to config contents.
+
+        The _ses_folders contains the entire directory tree for each data type. The structure is that the top-level folder (e.g. ephys,
+        behav, microscopy) are found in the project root. Then sub- and ses- folders are created in this project root, and all subfolders
+        are created at the session level.
+
+        TODO: - for adding new files, first this tree but be extended, then a new option added to inputs of make_config file. This can be
+                refactored and an easier way to indicate the relationship between folders found.
+              - factor out _ses_folders generation
+              - decide whether to repeat top-level dir name in the session level dir
+        """
+        self._ssh_key_path = self._join("appdir", self.username + "_ssh_key")
+        self._hostkeys = self._join("appdir", "hostkeys")
+
+        self._ses_folders = {
+            "ephys": Folder(
+                "ephys",
+                self.cfg["use_ephys"],
+                subfolders={
+                    "ephys_behav": Folder(
+                        "behav",
+                        self.cfg["use_ephys_behav"],
+                        subfolders={
+                            "ephys_behav_camera": Folder(
+                                "camera",
+                                self.cfg["use_ephys_behav_camera"],
+                            ),
+                        },
+                    ),
+                },
+            ),
+            "behav": Folder(
+                "behav",
+                self.cfg["use_behav"],
+                subfolders={
+                    "behav_camera": Folder("camera", self.cfg["use_behav_camera"]),
+                },
+            ),
+            "microscopy": Folder(
```

Comment:
As discussed, `microscopy` should be split into something like `imaging` (meaning e.g. 2p/widefield) and `histology`.